src/mlx/warnings.py

```diff
@@ -4,35 +4,43 @@
 from __future__ import print_function
 
 import argparse
+import errno
+import glob
 import json
-import os
 import subprocess
 import sys
-import glob
-from mlx.warnings_checker import SphinxChecker, DoxyChecker, JUnitChecker, XMLRunnerChecker, CoverityChecker
+
```

Comment:
Try to avoid `git commit -a` and in cases like this you can use `git add -i` to "interactively" add certain chunks as/when they are needed.